DigitalLearningSolutions.Data.Tests/Services/CourseServiceTests.cs

```diff
@@ -167,10 +176,10 @@ public void
         {
             // Given
             A.CallTo(() => courseDataService.GetCourseValidationDetails(A<int>._))
-                .Returns((2, 2));
+                .Returns((2, 2, false));
 
             // When
-            var result = courseService.VerifyAdminUserCanAccessCourse(1, 2, 0);
+            var result = courseService.VerifyAdminUserCanManageCourse(1, 2, 0);

```

Comment:
Test name `VerifyAdminUserCanAccessCourse_should_return_true_when_centreId_matches_and_admin_category_id_is_zero`
should change to match the new target of `VerifyAdminUserCanManageCourse`:
`VerifyAdminUserCanManageCourse_should_return_true_when_centreId_matches_and_admin_category_id_is_zero`